test/suites/dev/archive/archive-precompile-batch-gas-limit.ts

```diff
@@ -0,0 +1,62 @@
+import "@moonbeam-network/api-augment";
```

Comment:
What are archive tests ?

Reply:
I consider it a TODO list of test cases to go about reimplementing/reinstating.